ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/table-viz.view.js

```diff
@@ -15,17 +15,41 @@
 
 import ExportResults from '../../../react-component/container/table-export'
 import React from 'react'
+import { union } from 'lodash'
 const lightboxInstance = require('../../lightbox/lightbox.view.instance.js')
 import {
   Button,
   buttonTypeEnum,
 } from '../../../react-component/presentation/button'
-
 const Marionette = require('marionette')
+const Backbone = require('backbone')
 const CustomElements = require('../../../js/CustomElements.js')
 const TableVisibility = require('./table-visibility.view')
 const TableRearrange = require('./table-rearrange.view')
 const ResultsTableView = require('../../table/results/table-results.view.js')
+const ResultForm = require('../../../component/result-form/result-form.js')
+
+const filterAttributesWithResultForm = (
+  attributes,
+  resultTemplateAttributes
+) => {
+  const defaultAttributes = [
+    'id',
+    'title',
+    'metacard-type',
+    'metacard-tags',
+    'source-id',
+  ]
+  const filteredAttributes = attributes.filter(attribute =>
```

Comment:
```suggestion
  const filteredAttributes = attributes.filter(resultTemplateAttributes.includes)
```
✏️

Reply:
what is this function shorthand concept called?